### ISSUE TITLE
IE11 error when having a select-tag with name

### ISSUE THREAD

**Issue by rejas** [jimmywarting/FormData#45]
I am getting this error on the console when using v3.0.7 in IE11 

SCRIPT5007: Array.prototype.slice: 'this' is Null or undefined
formdata.min.js (4,1)

Example code is:

```
                            <select name="example">
                                <option value="volvo">Volvo</option>
                                <option value="saab">Saab</option>
                                <option value="mercedes">Mercedes</option>
                                <option value="audi">Audi</option>
                            </select>
```

If i remove the name-tag all is fine but then the select box doesnt show up in the final formdata :-(

**Comment by jimmywarting** [comment]
A fix is on the way.

For references: https://stackoverflow.com/questions/10711767/is-selectedoptions-broken-or

**Comment by jimmywarting** [comment]
fixed in v3.0.8

**Comment by rejas** [comment]
Awesome once again! Seems to work for me, probably in production too :-)
